enginetest/engine_only_test.go

```diff
@@ -667,6 +667,10 @@ func TestTableFunctions(t *testing.T) {
 			Query:    "select * from sequence_table('x', 5)",
 			Expected: []sql.Row{{0}, {1}, {2}, {3}, {4}},
 		},
+		{
```

Comment:
need a bit more testing :) take a stab at some of the critical paths where table aliases need to behave